web/src/pages/home/homePage.js

```diff
@@ -559,7 +525,9 @@ const Homepage = () => {
               <li>Manage everything centrally.</li>
               <li>Increase your booking metrics.</li>
             </ul>
-            <button className="list-property-button">List your property</button>
+            <a href="https://www.domits.com/landing" rel="noopener noreferrer">
```

Comment:
Dit moet global zijn, niet specifiek naar domits.com
```suggestion
            <a href="https://www.domits.com/landing" rel="noopener noreferrer">
```